app/__init__.py

```diff
@@ -7,8 +7,8 @@
 from flask import abort
 from flask import request
 
-from app.helpers import make_error_msg
 from app.helpers import ALLOWED_DOMAINS_PATTERN
+from app.helpers import make_error_msg
```

Comment:
Moves helpers in python files, e.g. `app.helpers.utils.py` in order to keep the `__init__.py` empty. Exception for the `app.__init__.py` which contains the Flask application.

Reply:
Good point, just wanted to be consistent with `service-shortlink`, `service-qrcode` and `service-icons` here.